thirdparty/openapi3/schemas/AuthorizationPostRequest.yaml

```diff
@@ -0,0 +1,26 @@
+title: AuthorizationsPostRequest
+description: POST /authorizations request object.
+type: object
+properties:
+  toParticipantId:
```

Comment:
at Switch there is no `toParticitpantId` (this field only is present at scheme endpoint) -> for this purposes the header is used

Reply:
fixed